plugins/casutg/src/main/java/com/blazemeter/jmeter/threads/concurrency/ConcurrencyThreadStarter.java

```diff
@@ -1,31 +1,49 @@
 package com.blazemeter.jmeter.threads.concurrency;
 
-import com.blazemeter.jmeter.threads.AbstractThreadStarter;
-import com.blazemeter.jmeter.threads.DynamicThread;
 import org.apache.jmeter.engine.StandardJMeterEngine;
 import org.apache.jmeter.threads.ListenerNotifier;
 import org.apache.jmeter.util.JMeterUtils;
 import org.apache.jorphan.collections.ListedHashTree;
 import org.apache.jorphan.logging.LoggingManager;
 import org.apache.log.Logger;
 
+import com.blazemeter.jmeter.threads.AbstractThreadStarter;
+import com.blazemeter.jmeter.threads.DynamicThread;
+
 public class ConcurrencyThreadStarter extends AbstractThreadStarter {
     private static final Logger log = LoggingManager.getLoggerForClass();
+    private static final long DEFAULT_SHIFT_RAMPUP = JMeterUtils.getPropDefault("dynamic_tg.shift_rampup_start", 0L);
```

Comment:
This is one of features to be able to change it in runtime via property. It can't be static, but can be cached like `rampUp`.

Reply:
Will be in next commit